cognite/client/_api/datapoints.py

```diff
@@ -959,27 +965,189 @@ def retrieve_dataframe(
+            granularity (str): The granularity to fetch aggregates at supported second, minute, hour, day, week, month, quarter and year. Default: None.
+            ignore_unknown_ids (bool): Whether to ignore missing time series rather than raising an exception. Default: False
+            uniform_index (bool): If querying aggregates and not use variable unit lengths (month, quarter, and year), specifying `uniform_index=True` will return a dataframe with an
+                equidistant datetime index from the earliest `start` to the latest `end` (missing values will be NaNs). If these requirements are not met, a ValueError is raised. Default: False
+            include_aggregate_name (bool): Include 'aggregate' in the column name, e.g. `my-ts|average`. Ignored for raw time series. Default: True
+            include_granularity_name (bool): Include 'granularity' in the column name, e.g. `my-ts|12h`. Added after 'aggregate' when present. Ignored for raw time series. Default: False
+            column_names ("id" | "external_id"): Use either ids or external ids as column names. Time series missing external id will use id as backup. Default: "external_id"
+
+        Returns:
+            pandas.DataFrame: A pandas DataFrame containing the requested time series with a DateTimeIndex localized
+            to the time zone from the start time.
+
+        Examples:
+
+            Get a pandas dataframe in the local timezone in Oslo, Norway:
+
+                >>> from cognite.client import CogniteClient
+                >>> try:
+                ...     from zoneinfo import ZoneInfo
+                ... except ModuleNotFoundError:
+                ...     from backports.zoneinfo import ZoneInfo
+                >>> client = CogniteClient()
+                >>> df = client.time_series.data.retrieve_dataframe_in_tz(
+                ...     id=12345,
+                ...     start=datetime(2023, 1, 1, tzinfo=ZoneInfo("Europe/Oslo")),
+                ...     end=datetime(2023, 2, 1, tzinfo=ZoneInfo("Europe/Oslo")),
+                ...     aggregates="average",
+                ...     granularity="1week",
+                ...     column_names="id")
+
+            Get a pandas dataframe with the sum of the time series with external id "foo" and "bar",
+            for each quarter from 2020 to 2022 returned in the local time zone Oslo, Norway.
+
+                >>> from cognite.client import CogniteClient
+                >>> from datetime import datetime
+                >>> try:
+                ...     from zoneinfo import ZoneInfo
+                ... except ModuleNotFoundError:
+                ...     from backports.zoneinfo import ZoneInfo
+                >>> client = CogniteClient()
+                >>> df = client.time_series.data.retrieve_dataframe(
+                ...     external_id=["foo", "bar"],
+                ...     aggregates=["sum"],
```

Comment:
Add an additional aggregate to further illustrate when to give a list vs a string directly? Would love one of the aggregates with an underscore 😉